### ISSUE TITLE
TX frequency calibration

### ISSUE THREAD

**Issue by degaus72** [df8oe/mchf-github#1115]
Hello,

mcHF v0.5 - bootloader 3.4.1 - firmware 2.5.118

While I was calibrating the TX frequency, I have noticed some discrepances between bands.
I have started to adjust the frequency on 80m following the manual and that was ok with a ppm of +10,4, but when I test on 40m freq. 7.184.000 the counter shows 7.183.249, then if I move to 20m freq. 14.244.000 it shows 14.243.997.

The 3Hz difference on 20m maybe be acceptable but the the 751Hz on 40m probabily not. I did try to adjust the 20m band but it messed up the 80m, am I missing some steps?

73 de M0TKM

**Comment by db4ple** [comment]
Hi,

I am sure you had CW on 40m, this causes your perceived deviation. Use the same mode on all bands gives consistent frequency measurements.

73
Danilo

**Comment by degaus72** [comment]
Hi Danilo,
I have the same settings in all the 3 bands.
I have attached some pictures.
73, Matteo
 

    On Thursday, October 5, 2017, 4:47:15 PM GMT+1, db4ple <notifications@github.com> wrote:  
 
 Hi,

I am sure you had CW on 40m, this causes your perceived deviation. Use the same mode on all bands gives consistent frequency measurements.

73
Danilo

**Comment by db4ple** [comment]
You should respond through the Webinterface, I can't see the pictures. And when responding via mail, please remove the text you respond to (otherwise the comments will be cluttered with comment quotes).
TU
Danilo

**Comment by degaus72** [comment]
Hi Danilo,

Sorry about that. First time I use the github webinterface.
Pictures attached....hope :-)

73, Matteo

![3763 freq](https://user-images.githubusercontent.com/30686993/31239273-74a11cd2-a9f4-11e7-8154-f49ddc1e9ccc.jpg)
![3763 mchf](https://user-images.githubusercontent.com/30686993/31239276-74a4befa-a9f4-11e7-86b6-f2bdc250d28c.jpg)
![7184 freq](https://user-images.githubusercontent.com/30686993/31239275-74a36334-a9f4-11e7-9f44-ef4a4cbc52bf.jpg)
![7184 mchf](https://user-images.githubusercontent.com/30686993/31239272-74a0c958-a9f4-11e7-8096-d6e0d41a8e41.jpg)
![14244 freq](https://user-images.githubusercontent.com/30686993/31239277-74a8f0c4-a9f4-11e7-8ec8-d0bb329a0c57.jpg)
![14244 mchf](https://user-images.githubusercontent.com/30686993/31239274-74a20296-a9f4-11e7-838c-0744a2538194.jpg)

**Comment by df8oe** [comment]
You are transmitting on 40m in LSB. TUNE uses sidetone: voilá - there is your offset!

You must use FM on ALL bands.

**Comment by s53dz** [comment]
I could not reproduce this issue.

My recent results on mcHF rev04, D2.5.118 are like this:
```
dialed          measured
29.000.000 - 29.000.008
21.100.000 - 21.100.007
14.100.000 - 14.100.004
07.100.000 - 07.100.001
03.600.000 - 03.599.999
```

And also tried the above frequencies.
Measured with a frequency counter synchronized to a 10 MHz OCXO frequency standard.
OCXO calibrated with GPS-DO.
TCXO temp = 35.9°
RF power = 1W

73 Bojan

**ADDED:** Yes, LSB on 40m. I did not see your pictures when writing this.
I used the PTT instead.
